### PR TITLE
bug(folder-slice): fix delete folder bug 

### LINKED ISSUE #47: Unable to edit or delete newly created folders
**Is your feature request related to a problem? Please describe.**
New folders are not able to be deleted or edited

**Describe the solution you'd like**
New folders should be able to be edited and deleted.

### PR DESCRIPTION
**Pull request by dannysanchez559**
## Changes
_List Changes Introduced by this PR_
1. Fix delete folder button functionality

## Purpose
For delete button inside edit folder modal to work properly

Closes #47

### CODE REVIEW

File: redux/folderSlice.js [hunk 1]
```diff
@@ -116,6 +117,8 @@ const folderSlice = createSlice({
       delete state.allFolder[folder.name];
       const updatedFolder = Object.assign(folderToUpdate, updatedValues);
       state.allFolder[updatedFolder.name] = updatedFolder;
+
+      console.log("updatedFolder: ", updatedFolder);
```
Comment (BlackCubes) [inline redux/folderSlice.js:121]: Wat's dis???????????

Comment (connielion) [inline redux/folderSlice.js]: Do you need this?

**Review by BlackCubes — APPROVED** [review]
Console.log!!!!!!!

File: redux/folderSlice.js (hunk 1 above)
Comment (Lap343) [inline redux/folderSlice.js]: you want to log this?

**Review by connielion — APPROVED** [review]
Nice and short, me likey!

**Review by Lap343 — APPROVED** [review]
never mind :)

**Pull request merged**